### PR TITLE
Added upcoming badge, other UI tweaks

### LINKED ISSUE #233: Remove Like Button for MVP
# Description
We aren't going to get to adding likes to Penny Chats for the MVP, so we should remove the button from the view.

### LINKED ISSUE #238: Add "Upcoming" badge to Penny Chats that haven't happened yet
# Description
We need a simple way to call out upcoming vs past Penny Chats in the Penny Chat list.

## Details
- [x] Add a badge to the Penny Chats in the list that haven't happened yet

### PR DESCRIPTION
**Pull request by nchouard**
# Description
Adds an **Upcoming** badge to the chat list cards to make it clear what Penny Chats are happening in the future. Closes #238 

## Details
In addition to adding the Upcoming badge, this change includes a few other UI tweaks: 
* Removed heart button from Chat details (closes #233)
* A container was added to the navbar to match our content (and our site's homepage). 
* The locale time was added to Penny Chats that are upcoming.
* The chat detail headers were being capitalized by some bootstrap styles, so that style was overriden.

Updated UI

### CODE REVIEW

File: penny_university_frontend/src/components/Date.tsx [hunk 1]
```diff
@@ -6,8 +6,11 @@ type DateProps = {
   className: string,
 }
 
-const Date = ({ date, className }: DateProps) => (
-  <p className={className}><strong>{moment(date).format('M/D/YYYY')}</strong></p>
-)
+const Date = ({ date, className }: DateProps) => {
+  let format = moment(date) > moment() ? 'M/D/YYYY @ h:mm A' : 'M/D/YYYY'
```
Comment (CoreyAR) [inline penny_university_frontend/src/components/Date.tsx:10]: I mean to remove this component and add the date formatting to the model. We could also the upcoming logic to the model.

File: penny_university_frontend/src/components/chats/ChatCard.tsx [hunk 2]
```diff
@@ -17,8 +19,9 @@ type ChatCardProps = {
 const ChatCard = ({ chat }: ChatCardProps) =>
   chat ? (
     <Card body className="mb-3 border-0 shadow-sm">
-      <CardTitle tag="h5">
+      <CardTitle tag="h5" className="d-flex">
         <Link className="text-reset" to={`/chats/${chat.id}`}>{chat.title}</Link>
+        {moment(chat.date) > moment() ? <Badge className="ml-2" color="danger">Upcoming</Badge> : null}
```
Comment (CoreyAR) [inline penny_university_frontend/src/components/chats/ChatCard.tsx:24]: We can consolidate this logic on the chat model

**Comment by nchouard** [comment]
@CoreyAR this looks good, just need to fix the merge conflicts!

**Comment by CoreyAR** [comment]
👍 Sounds good. I approved the PR.

**Pull request merged**